### ISSUE TITLE
Crash Report -548267435

### ISSUE THREAD

**Issue by NuptupTDOW** [wowsims/wotlk#2615]
Imported a guildie's character and was trying to run the sim and this happened. Any idea what's going on? Here's the import string:

{"talents":"2315020530033303201023001350-00000000000000000000000000000-2302300050230100000000000000000","glyphs":{"major":["Glyph of Death Strike","Glyph of Disease","Glyph of Dark Death"],"minor":["Glyph of Raise Dead","Glyph of Horn of Winter","Glyph of Death's Embrace"]},"class":"deathknight","race":"Human","name":"Grïmreaper","gear":{"items":[{"id":44695,"gems":[]},{"id":40369,"gems":[]},{"id":40185,"gems":[]},{"id":40250,"gems":[]},{"enchant":1891,"gems":[39999,40014],"id":40781},{"enchant":1600,"gems":[],"id":40733},{"enchant":1603,"gems":[],"id":41349},{"id":37178,"gems":[]},{"id":37193,"gems":[39996,40014]},{"enchant":1597,"gems":[],"id":37068},{"id":37819,"gems":[]},{"id":42112,"gems":[]},{"id":42131,"gems":[]},{"id":38674,"gems":[]},{"enchant":3368,"gems":[],"id":41816},null,{"id":40715,"gems":[]}]},"professions":[{"name":"Enchanting","level":19},{"name":"Herbalism","level":450}],"level":80,"spec":"unholy","realm":"Pagle"}

<<<Automated Crash Report Below This Line>>>

Link:
https://wowsims.github.io/wotlk/deathknight/#eJzjYtFgzGB0YhBikFrI2sTIFpCTWJlaxMEqwCXVxsjFLnBsIZMSIwMXi8DG3UxA7s+dUO6uXUxcPAJn9zEpse3fwXRuBxMXl8DcfUwSB3hA8lwCrYeZJA6D2RwCuxYxKTExMADVnwSy2PZA1LMLnFnIJLGXB2hYw0kmLpChk8E0u8CNY0wSK6TAQt1AGzg5mAx4Kxg7GBkXMZ1l4poBNGsUYANCBqTqqGL4xajGJSrA6MEYwJgIFnLod9jByHiCkVGKVxCi6IO9AqMWQxdjlJGxoamBkYGpsYGBsbGxgbGRgaGBEZBjaGxqoGuAD+gClYFUGpiCKEMM+UeMEhybrzEJvH3KJPHmKZPCzKtMGjOuMBlMvsL0gpF1FiPDLUaGWUwMSo+ZOMFOakhzgDpOxEFy1kwQOGlvCRG5YK+YBgbX7I02MykES7Bq0TjJfGh0oIYxDSePUMUcUgAkzPa+d3CE8H/YJzAGgTPZ4a8aMf2HOB2uLy6w5br+2J5YEyGx8dIeov8qDn0f7HfItb4O3PHPPgjMeGEP0ffXfgnIusUv7EXWuT+sEvmE095qsIIWot1FLSBkcPYMCHA4QFz6h6ALAPLbnb0=

RNG Seed: 3055115118

runtime error: invalid memory address or nil pointer dereference
Stack Trace:
goroutine 9 [running]:
runtime/debug.Stack()
	/opt/hostedtoolcache/go/1.18.10/x64/src/runtime/debug/stack.go:24 +0x6
github.com/wowsims/wotlk/sim/core.runSim.func1()
	/home/runner/work/wotlk/wotlk/sim/core/sim.go:59 +0x10
panic({0xf0940, 0x135d920})
	/opt/hostedtoolcache/go/1.18.10/x64/src/runtime/panic.go:838 +0x29
github.com/wowsims/wotlk/sim/deathknight.(*Deathknight).drwCountActiveDiseases(...)
	/home/runner/work/wotlk/wotlk/sim/deathknight/diseases.go:12
github.com/wowsims/wotlk/sim/deathknight.(*Deathknight).newHeartStrikeSpell.func1(0x21ee6e0, 0x142b900, 0x22f4b40)
	/home/runner/work/wotlk/wotlk/sim/deathknight/heart_strike.go:48 +0x16
github.com/wowsims/wotlk/sim/core.(*Spell).applyEffects(0x22f4b40, 0x21ee6e0, 0x142b900)
	/home/runner/work/wotlk/wotlk/sim/core/spell.go:503 +0x23
github.com/wowsims/wotlk/sim

**Comment by NuptupTDOW** [comment]
I tried moving a talent point around, removing it from unholy blight to put it back into dancing rune weapon and the sim ran correctly. As soon as Dancing Rune Weapon is removed, the sim breaks and throws that crash report. This was fixed before in #2560 and was working fine over the weekend but seems to not work now. I have restored all defaults, and am using the internal preset blood spec with P! default gear and as soon as I remove DRW, if fails. Previously, in the mentioned error report it would still run but would be wildly inaccurate, but now it just crashes.
